### PR TITLE
change granularity for copy pipe

### PR DESCRIPTION
**Pull request by jlmadurga**
I've tried to simulate some real iteration of changing granularity:
* new datasource
* new pipe + new scheduling
Requires backfilling, done with MV (copy pipes limit is 1 so can't be done with copy pipe)

### CODE REVIEW

**Comment by xavijam** [comment]
Looks amazing 🎉, but I miss the `README.md` explaining how to do it. Check other finished use-cases 🙏

**Comment by jlmadurga** [comment]
> Looks amazing 🎉, but I miss the `README.md` explaining how to do it. Check other finished use-cases 🙏

It's WIP. Still need to finish the backfill that I will need a one-off copy

**Comment by xavijam** [comment]
> Not sure if we are following some README format. For sure README needs some review

Reviewing it!

File: change_copy_pipe/README.md [hunk 1]
```diff
@@ -1,3 +1,107 @@
-# Change Copy Pipe changing target Data Source
+# Tinybird Versions - Change copy pipe to change time granularity
+
+
+
+We used copy pipe `bots_snapshot` that is scheduled to every hour do snapshot of bots hits on target `copy_bots_snapshot` Data Source.
+
+We decide we want to change time granularity to day. We want to:
```
Comment (xavijam) [inline change_copy_pipe/README.md:7]:
```suggestion
We want to change time granularity to day. So, we need:
```

File: change_copy_pipe/README.md [hunk 2]
```diff
@@ -1,3 +1,107 @@
...
+    WHERE toDate(date_hour) < today()
+    GROUP BY date, pathname, bot_source
+
+TYPE materialized
+DATASOURCE copy_bots_snapshot_v1
+```
+
+In this case we are stopping backfill on today. Remember in this example we are calculating yesterday snapshot.
+
+## Step 5: Validate changes on CI
```
Comment (xavijam) [inline change_copy_pipe/README.md:83]:
```suggestion
## Step 4: Validate changes on CI
```

File: change_copy_pipe/README.md [hunk 3]
```diff
@@ -1,3 +1,107 @@
...
+> By default uses `tb deploy --populate --fixtures --wait` so we can use Materialize View populate as backfill
+
+Then validate changes updating tests:
+
+- fixtures tests: update tests but new fixtures are not need it as we rely on backfill to have desired data in new version
+- data quality tests: almost same validations but now asserting daily dates
+
+> Better to rely on backfill instead of generating new static fixtures so we can validate backfill before deploying
+
+## Step 6: Deploy to production
```
Comment (xavijam) [inline change_copy_pipe/README.md:96]:
```suggestion
## Step 5: Deploy to production
```

**Review by xavijam — COMMENTED** [review]
Looking great, just a few comments

**Pull request merged**